src/js/filters/filters.filter.js

```diff
@@ -274,6 +274,16 @@ module.filter("rpripplename", ['$rootScope', '$http', 'rpId', function($scope, $
   }
 }]);
 
+module.filter("rpripplenamefull", ['$rootScope', '$http', 'rpId', function($scope, $http, id) {
```

Comment:
I know you just copied this, but you don't use `$scope` and `$http` you can take them off